src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -387,4 +387,7 @@
   <data name="PX1096" xml:space="preserve">
     <value>PXOverrideSignatureMismatch</value>
   </data>
+  <data name="PX1007IncorrectProjectionDacFieldComment" xml:space="preserve">
```

Comment:
@EAndrosova  please review the short name for new PX1007 diagnostic on projection DAC property incorrect comment